alf/utils/data_buffer.py

```diff
@@ -133,6 +133,10 @@ def _create_buffer(tensor_spec):
         if allow_multiprocess:
             self.share_memory()
 
+    def circular(self, t):
```

Comment:
prefix an '_' as it's private use.

Reply:
It's used by subclass as well.

Reply:
It's fine to have '_' even for subclass.